gradle-lombok-plugin/src/main/groovy/io/franzbecker/gradle/lombok/task/VerifyLombokTask.groovy

```diff
@@ -23,7 +23,7 @@ class VerifyLombokTask extends DefaultTask {
     void verifyLombok() {
         // Retrieve extension and configuration
         def extension = project.extensions.findByType(LombokPluginExtension)
-        def configuration = project.configurations.getByName(LombokPlugin.LOMBOK_CONFIGURATION_NAME)
+        def configuration = project.configurations.getByName(LombokPlugin.VERIFY_LOMBOK_CLASSPATH_CONFIGURATION_NAME)
```

Comment:
I might be too far away from this code right now, but this doesn't seem right.
Just changing the way we retrieve the configuration without changing the way the Lombok dependency is added doesn't work in my head 🙈

Reply:
Heheh, no worries. :slightly_smiling_face: The reason this works is because the `compileClasspath` extends from `compileOnly`, but `compileClasspath` is _resolvable_ (listed under "configurations used by consumers") while `compileOnly` is not (listed under "configurations used to declare dependencies"). See: https://docs.gradle.org/7.2/userguide/java_library_plugin.html#sec:java_library_configurations_graph

Reply:
Ah okay – thanks and sorry for being so slow in responding!